rules_building_block/defense_evasion_masquerading_communication_apps.toml

```diff
@@ -21,7 +21,7 @@ name = "Potential Masquerading as Communication Apps"
 risk_score = 21
 rule_id = "c9482bfa-a553-4226-8ea2-4959bd4f7923"
 severity = "low"
-tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion"]
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR"]
```

Comment:
Calling the rule type `BBR` is misleading, since the rule type is actually defined under `type` (`eql` in this case). To avoid confusion and overloading the term, we should just tag it with `building_block` or `building_block: true`

Reply:
This verbiage was defined as part of the tag reform, as we have `Rule Type: Higher-Order Rule`, `Rule Type: Indicator Match`, `Rule Type: ML`, etc